src/main/shadow/cljs/devtools/server/web.clj

```diff
@@ -219,6 +219,7 @@
           [:script "shadow.test.workspaces.init();"]
           ])})))
 
+
```

Comment:
What are these? :P

Reply:
don't know. I guess I was fixing the printer thing and they didn't notice